Services/Notifications/classes/class.ilNotificationGUI.php

```diff
@@ -103,12 +78,12 @@ public function getHandler($type)
         return $this->handler[$type];
     }
 
-    private function getAvailableTypes($types = array())
+    private function getAvailableTypes($types = array()) : array
```

Comment:
Please add the type hint and specify the type of the array elements (return type) by using PHPDoc.